reqs/README.md

```diff
@@ -47,7 +45,6 @@ Requirements
     - reqs/requirements-dev-spark.txt
     - reqs/requirements-dev-azure.txt
     - reqs/requirements-dev-contrib.txt
-    - reqs/requirements-dev-cloud.txt
```

Comment:
This is still pulled in during type checking for the orjson dependency. Could you add this line back here?

Reply:
thanks for catching - re-added